### DISCUSSION TITLE
old version agents talking to new collectors

### DISCUSSION THREAD

**Discussion by knitehias** [jaegertracing/jaeger#3802] [General]
Hello,

I would like to upgrade my jaeger but collector is the easiest part so can I upgrade collector first then agents gradually.
That means I will have 2 - 3 older version agents talking to a new version collector.

I know it is not best practise but do you this will be possible ;) ?

Best Regards

**Comment by yurishkuro** [comment]
There have never been any breaking changes in the protocol between agents and collector, so old agent versions will work fine.

**Reply by knitehias** [reply]
> There have never been any breaking changes in the protocol between agents and collector, so old agent versions will work fine.

Thanks a lot for the quick answer !
